### ISSUE TITLE
Death Knight Intro Quest: The Endless Hunger (Cannot move forward with DK in current state)

### ISSUE THREAD

**Issue by cloud1121** [SallyWhitemane/Herod#581]
[//]: # (REMBEMBER! Add links to things related to the bug using for example:)
[//]: # (http://wowhead.com/)
[//]: # (cata-twinhead.twinstar.cz)

**Description:**
The Endless hunger quest item is not usable on any NPC in the arena for the quest. 
The quest is currently incompletable and prevents progression in DK questline
https://streamable.com/wv2rj4
https://streamable.com/flxsuo

**How to reproduce:**
Make new dk, do intro quests, it is the 3rd quest into the chain

**How it should work:**
Should be able to click the soul prisons with the key to start the event

**Database links:**

**Comment by cloud1121** [comment]
@HerodDev possible to escalate this since it is keeping any new DK's hard stuck from moving further?

**Comment by HerodDev** [comment]
It is working, just click on the button on wall
![image](https://user-images.githubusercontent.com/82453199/120100672-34a28300-c142-11eb-80d6-74e49cfab250.png)

**Comment by cloud1121** [comment]
@HerodDev still not working live side. Both Ahumaya and I tested.

**Comment by cloud1121** [comment]
![image](https://user-images.githubusercontent.com/77916637/120112709-be932180-c12b-11eb-9356-593975174fe3.png)

**Comment by Ahumaya** [comment]
Tested on my Orc DK "Go", the skull objects were not intractable, either through normal right click or via item interaction. I dropped the quest and required, but was unable to get the interaction to work.

**Comment by Ahumaya** [comment]
Tested again today on Go, the skulls do work, however the Unworthy Initiate does not engage in combat with the player, shown in the video below.

https://user-images.githubusercontent.com/82483575/120251049-e1195d80-c245-11eb-8aa8-1ea189a288f5.mp4


I also observed that on working skull features, chains appear between the skull and the Unworthy Initiate. In earlier screenshots, these chains were not present. I have noted that currently, if you click on a skull, the chains disappear and the Unworthy Initiate stands there, not doing anything, and the skull cannot be interacted with. I wonder if this issue caused the skulls to appear not to be interactable.
